src/index.ts

```diff
@@ -85,10 +85,14 @@ module.exports = {
             ...htmlParsers.vue,
             preprocess: vuePreprocessor,
         },
-        svelte: {
-            ...svelteParsers.parsers.svelte,
-            preprocess: sveltePreprocessor,
-        },
+        ...(!!svelteParsers.parsers
```

Comment:
```suggestion
        ...(svelteParsers.parsers
```

the ternary operator here works just fine like this.

Reply:
Done ✅